spec/services/document_converter_service_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe DocumentConverterService do
+  subject(:convert_document) { described_class.new(document, new_document).call }
```

Comment:
would this not be clear if it was
```suggestion
  subject(:convert_document) { described_class.new(document, converted_document).call }
```
or similar `converted_preview_document`, `pdf_document`, `preview_document`